src/plots/VolumeVTK9/avtVisItVTKRenderer.C

```diff
@@ -0,0 +1,782 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+
+// *************************************************************************
+//   avtVisItVTKRenderer.C
+// *************************************************************************
+
+#include <avtVisItVTKRenderer.h>
+
+#include <avtParallel.h>
+
+#include <DebugStream.h>
+#include <TimingsManager.h>
+#include <StackTimer.h>
+#include <ImproperUseException.h>
+#include <InvalidVariableException.h>
+#include <avtCallback.h>
+#include <avtDatasetExaminer.h>
+
+
+#include <vtkImageData.h>
+#include <vtkCellData.h>
+#include <vtkPointData.h>
+#include <vtkDataArray.h>
+#include <vtkFloatArray.h>
+#include <vtkDataObject.h>
+#include <vtkRectilinearGrid.h>
+#include <vtkRenderer.h>
+#include <vtkVolume.h>
+#include <vtkVolumeProperty.h>
+#include <vtkGPUVolumeRayCastMapper.h>
+#include <vtkSmartVolumeMapper.h>
+#include <vtkColorTransferFunction.h>
+#include <vtkPiecewiseFunction.h>
+
+#include <visit-config.h> // for HAVE_OSPRAY
+
+#ifdef HAVE_OSPRAY
+    #include <vtkOSPRayVolumeMapper.h>
+    #include <vtkOSPRayRendererNode.h>
+#endif
+
+#include <string>
+#include <vector>
+
+// #define DUMP_IMAGE_DATA
+
+#ifdef DUMP_IMAGE_DATA
+    #include <vtkXMLImageDataWriter.h>
+#endif
+
+// #ifdef PARALLEL
+//   #define LOCAL_DEBUG std::cerr << __LINE__ << " [VisItVTKRenderer] " \
```

Comment:
Other than going to `std:cerr`, I dunno what `LOCAL_DEBUG` is better for over VisIt's own `debugX` log files.

Perhaps we need a way to stream `debugX` to `std::cerr`? We do support UNbuffered streams to our debug logs (I think by default) and so `tail -f <log-file-name>` *while* VisIt is running works very much similarly to going direct to `std::cerr`.

Because our log files are named with MPI rank, we don't then prepend MPI rank to error messages in those files and that is something new here also I guess.

Maybe when `debugX` is instead stream to `std::cerr` or `std::cout`, we prepend rank to messages too.

Anyways, I would remove these now. They were obviously for debugging support durring development and perhaps mostly useful only to the developer, @ARSanderson. 

Or, better yet...

```
#ifdef FOR_ARSANDERSON
    #ifdef PARALLEL   
        #define LOCAL_DEBUG ...
    #else
        #define LOCAL_DEBUG ...
     #endif
#else
    #define LOCAL_DEBUG debug5     
#endif
```
kdding about the first part but